rpm/sailfish-secrets.spec

```diff
@@ -225,6 +225,9 @@ BuildRequires:  pkgconfig(Qt5DBus)
 BuildRequires:  pkgconfig(dbus-1)
 BuildRequires:  pkgconfig(libcrypto)
 BuildRequires:  qt5-plugin-sqldriver-sqlite
+BuildRequires:  pkgconfig(libgpg-error)
```

Comment:
That's wise. Changed for this.

Reply:
After more thought, is there any way to implement the pinentry component such that it can be licensed with BSD-3-clause also?  Given that we were hoping to distribute this gnupg plugin along with the daemon, I'd prefer a maximally permissive license to avoid potential inadvertent violations...

Reply:
I understand, but the pinentry program is not linking with the daemon. In fact, it ideally should be part of GnuPG itself which is GPL, like other pinentries. Indeed, the pinentry program is used by GnuPG executable to ask for user entry and is using IPC (via libassuan which is LGPL) to interface with GnuPG. So it may even be whatever license.

Our pinentry implementation is then linking with libsecret to interface nicely with SFOS environment. But since libsecret is BSD licensed, the pinentry can be whatever restrictive license we want.

So I think putting the pinentry under GPL won't violate anything. If you think differently, I can change it for BSD though.